server/utils/queryHelpers/scopeGet.ts

```diff
@@ -328,6 +328,19 @@ getActivePermissions = async (
 		canAdminCommunity ||
 		scopeMemberData.find((member) => member.communityId && member.permissions === 'manage');
 
+	const canEditCommunity =
+		isSuperAdmin ||
```

Comment:
These checks are redundant — if `isSuperAdmin || canAdminCommutity,` then `canManageCommunity` will be true.